src/syntax/platform.rkt

```diff
@@ -175,11 +152,9 @@
     (warn 'platform
           "platform has missing optional implementations: ~a"
           (string-join (for/list ([m (in-set missing)])
-                         (match-define (list name itypes ... otype) m)
-                         (format "(~a ~a)" name `(,@itypes ,otype)))
+                         (format "~a" m))
                        " ")))
-  ; make the platform
-  (create-platform #f reprs (append convs impls) (make-immutable-hash (hash->list costs)) (hash)))
+  (create-platform #f reprs impls (make-immutable-hash (hash->list costs)) repr-costs))
 
 (begin-for-syntax
 
```

Comment:
There may be some unused functions in `begin-for-syntax` blocks. Magic Racket should tell you which ones are unused